src/test/java/gov/healthit/chpl/aqa/stepDefinitions/ChplDownloadSteps.java

```diff
@@ -258,36 +258,29 @@ public void theDownloadFileHasAtLeastItems(final String itemParam) {
     }
 
     /**
-     * Select Surveillance Activity file from drop down and download .csv file.
+     * Select Surveillance Activity file from drop-down.
      */
-    @When("^I download the Surveillance Activity file$")
-    public void downloadSurveillanceActivityFile() {
+    @When("^I select the Surveillance Activity file from drop-down$")
+    public void selectSurveillanceActivityFile() {
```

Comment:
All of these steps should be combined into one, with a parameter for the name of the file as it's shown in the download drop-down. The Page Object elements should be consolidated into a single one as well, where instead of going to the "nth" item in the select, it gets one based on the visible name